README.rst

```diff
@@ -0,0 +1,144 @@
+
+Simple Microservice Configuration
```

Comment:
I love this write up, just not sure it justifies the wholesame conversion to rst. This library doesn't change a lot and I don't forsee the need to add significantly more docs.